docs/production/config.rst

```diff
@@ -24,10 +31,16 @@ Then add the key to the SSH configuration file (``~/.ssh/config``), so it knows
 
 To use the SSH key on the server, you have to add the contents of the **public** key (e.g. ``/home/<username>/.ssh/pumpkin_server.pub``) to server's ``/home/discord/.ssh/authorized_keys``.
```

Comment:
I'd just link to this nice article by DigitalOcean or directly use its content:
https://www.digitalocean.com/community/tutorials/how-to-set-up-ssh-keys-2

It uses `ed25519` keys, nicely walks through the key creation and even shows the usage of `ssh-copy-id` which eliminates the need to manualy configure `authorized_keys` on the server side. Overall a nice tutorial IMHO.